src/components/IloadDialog.tsx

```diff
@@ -1,61 +1,61 @@
-import { useState } from 'react'
+import { useEffect, useState } from 'react'
 import Button from '@mui/material/Button'
 import Dialog from '@mui/material/Dialog'
 import DialogActions from '@mui/material/DialogActions'
 import DialogContent from '@mui/material/DialogContent'
 import DialogTitle from '@mui/material/DialogTitle'
-
-import axios, { AxiosResponse } from 'axios'
+import Autocomplete from '@mui/material/Autocomplete';
+import TextField from '@mui/material/TextField';
+import axios from 'axios'
 import { toast } from 'react-hot-toast'
-import { useNavigate } from 'react-router-dom'
+import { ColumnCandidates } from '../@types/view';
 
-type Response = {
-  is_text_column_ok: boolean
-  is_date_column_ok: boolean
+interface SuccessLoadingResponse {
+  success: string;
+}
+interface ErrorLoadingResponse {
+  error: string;
 }
+type LoadingResponse = SuccessLoadingResponse | ErrorLoadingResponse;
 
 interface Props {
   open: boolean
   setOpen: (value: boolean) => void
   handleViewing: any
+  columnCandidates: ColumnCandidates;
 }
 
-export default function IloadDialog({ open, setOpen, handleViewing }: Props) {
-  const [textValue, setTextValue] = useState('text')
-  const [dateValue, setDateValue] = useState('datetime')
+const labelStyle = { display: 'inline-block', width: '10em', marginRight: '0.5em' };
+const autocompleteStyle = { width: '15em' };
```

Comment:
I guess Tailwind CSS simply is unable to handle both this styles, am I right?

Reply:
The second one is not related to style prop rather it is used for sx prop of mui-component.
The first one can be handled by Tailwind CSS and was introduced mechanically by autocompleteStyle analogy. Can be modified